### ISSUE TITLE
multiplayer-fps subject

### ISSUE THREAD

**Issue by jo-eman** [01-edu/public#2416]
multiplayer-fps
---
The project instructions could be clarified. It suggests a "peer to peer" architecture in one place "Architecture
A peer-to-peer network that will allow other players to join your server and play against each other." but other parts of description suggest a more traditional "client server" architecture. 

Please clarify if project should be created with one server that many clients can connect to (then maybe rephrase the "peer to peer" part) or if the project really requires some peer to peer system where any client (peer) can join by connecting to any other client (then some other parts of isntructions need rephrasing)

**Comment by MSilva95** [comment]
Hello @jo-eman! Thanks for the feedback, we will change the peer to peer so that is more clear!
